assets/css/properties_panel/_status_run_block_tabs.scss

```diff
@@ -0,0 +1,75 @@
+.m-tabs {
+  flex: 1 1 auto;
+}
+
+.m-tabs__tabs {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.m-tabs__tab-list {
+  border-bottom: 1px solid $color-button-simple;
+  display: flex;
+  margin: 0;
+  margin-block-end: 0;
+  margin-block-start: 0;
+}
+
+.m-tabs__tab {
+  @include button-base;
+  background-color: $color-button-simple;
+  border-left: 1px solid $white;
+  border-radius: 0;
+  color: $color-font-grey;
+  display: inline-block;
+  flex: 1 0 auto;
+  font-weight: 900;
```

Comment:
We only load weights `400`, `500`, and `700`, so this is showing up as `700`. I think that's fine. `button-base` sets the weight to `700`, so this line doesn't end up doing anything, and could be removed.

Reply:
@ingridpierre Do you want the type weight on the tabs to be 900, or is 700 ok with you?

Reply:
700 looks alright in your screenshot. I was more concerned about emphasis when I was thinking about it as a brand new feature, less so now.